tests/tag-endpoints.test.js

```diff
@@ -0,0 +1,48 @@
+const request = require('supertest');
+const app = require('../server.js');
+// const connection = require('../db.js')
+
+describe('GET /tag', () => {
+  /* describe('Récupération de la liste des tags', () => {
+    let res;
+    const query = "INSERT INTO `tag` (`name`, `content`, `slug`) VALUES ('Javascript', 'JavaScript', 'js');";
+    beforeEach(done => connection.query(query); */
+
+  // Requête de la liste globale des tags
+  // Retourne le code 200 et un objet JSON contenant la liste des tags
+  it('returns 200 status', async () => {
+    return request(app)
+      .get('/tag')
+      .expect(200)
+      .expect('Content-Type', /json/)
+      .then(response => {
+        const expected = { data: [] };
+        expect(response.body).toEqual(expected);
+      });
+  });
+
+  // Requête pour récupérer un tag en particulier
+  // Retourne le code 200 et le contenu du tag en question
+  // ou le code 404 si l'id n'est pas attribué
+  it('returns 200 status if the id exists', async () => {
```

Comment:
Euh ??

Reply:
Commentaire pour deux 'it'... Je modifie ça.